Sources/FTAPIKit/URLServer+Async.swift

```diff
@@ -1,7 +1,7 @@
 #if swift(>=5.5)
```

Comment:
Nebylo by dobré ještě navýšit verzi swiftu na 5.5.1 a potom mít druhou variantu s verzí 5.5 a iOS 15? 🤔

Reply:
Ale vypadá, že by to nemusel být problém.

Reply:
Asi se mi to podaří, ale ne úplně elegantně, protože ten extension budu muset duplikovat.

Reply:
Otázka jestli to stojí za to. Nadruhou stranu tex XCode 13.0 může zlobit 😀. Další variantou by bylo to mít všechno jen až od swift 5.5.1 a nepodporovat to v XCode 13.0. 🤔